src/Blt/Plugin/Commands/SwsCommands.php

```diff
@@ -230,4 +232,126 @@ protected function getSiteAliases() {
     return $this->siteAliases;
   }
 
+  /**
+   * Create releases for all custom packages.
+   *
+   * @command sws:releases
+   *
+   * @option packages
+   *   Comma separated list of packages to create lists for.
+   */
+  public function swsReleases($options = ['packages' => '']) {
+    $docroot = $this->getConfigValue('docroot');
+    foreach (glob("$docroot/*/custom/*/*info.yml") as $package) {
+      $package_name = basename($package, '.info.yml');
+
+      if (!empty($options['packages'])) {
+        $packages = explode(',', $options['packages']);
+        if (!in_array($package_name, $packages)) {
+          continue;
+        }
+      }
+
+      $this->say(sprintf('Creating release for %s', $package_name));
+      $this->createRelease($package);
+    }
+  }
+
+  /**
+   * Edit the Changelog and create a pull request for a package.
+   *
+   * @param string $dir
+   */
+  protected function createRelease($info_path) {
+    $dir = dirname($info_path);
+    $base_branch = 'master';
+    $branch_name = $this->taskGit()
+      ->dir($dir)
+      ->exec('rev-parse --abbrev-ref HEAD')
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+    if ($branch_name == 'HEAD') {
+      throw new \Exception('Invalid branch name for ' . $dir);
+    }
+
+    $this->taskGit()->dir($dir)->pull('origin', $branch_name)->run();
+
+    $info = Yaml::decode(file_get_contents($info_path));
+    $version = str_replace('-dev', '', $info['version']);
+    $anchor = fgets(fopen("$dir/CHANGELOG.md", 'r'));
+
+    $log = $this->taskGit()
+      ->dir($dir)
+      ->exec("log --oneline origin/$base_branch..HEAD")
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+
+    $log = explode("\n", $log);
+    foreach ($log as $key => &$line) {
+      $line = preg_replace('/^.*? /', '- ', $line);
+      if (strpos(strtolower($line), 'back to dev') !== FALSE) {
+        unset($log[$key]);
+      }
+    }
+
+    if (empty($log)) {
+      return;
+    }
+    $header = [
+      $version,
+      str_repeat('-', 80),
+      '_Release Date: ' . date('Y-m-d') . '_',
+      PHP_EOL,
+    ];
+
+    $this->taskChangelog("$dir/CHANGELOG.md")
+      ->version($version)
+      ->setHeader(implode(PHP_EOL, $header))
+      ->setBody(implode("\n", $log) . PHP_EOL)
+      ->anchor($anchor)
+      ->run();
```

Comment:
This would be the right place to extract the changes for staci, et. al. if you just append them to a file somewhere convenient, yeah?